Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14.4.0-buster-slim
+FROM node:18.18.2-buster-slim
```

Comment:
We're currently updating all first-party actions to node20, do you think we should do that here too?